tests/scripts/install_python.bat

```diff
@@ -0,0 +1,8 @@
+SETLOCAL EnableDelayedExpansion
```

Comment:
Will it be possible to add some docs/headers about how this particular script works, what arguments are required and what requirements are needed?